libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat

```diff
@@ -0,0 +1,343 @@
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+IOMCU_UART USART6
+
+# Ethernet
+PC1 ETH_MDC              ETH1
+PA2 ETH_MDIO             ETH1
+PC4 ETH_RMII_RXD0        ETH1
+PC5 ETH_RMII_RXD1        ETH1
+PG13 ETH_RMII_TXD0       ETH1
+PG12 ETH_RMII_TXD1       ETH1
+PB11 ETH_RMII_TX_EN      ETH1
+PA7 ETH_RMII_CRS_DV      ETH1
+PA1 ETH_RMII_REF_CLK     ETH1
+#PG15 ETH_POWER_EN       ETH1
+
+PG15 Ethernet_PWR_EN OUTPUT HIGH # disable power on ethernet
+
+define BOARD_PHY_ID  MII_LAN8742A_ID
+define BOARD_PHY_RMII
+
+# ADC
+PA0 SCALED1_V3V3 ADC1 SCALE(2)
+PA4 SCALED2_V3V3 ADC1 SCALE(2)
+PB0 SCALED3_V3V3 ADC1 SCALE(2)
+PF12 SCALED4_V3V3 ADC1 SCALE(2)
+
+PB1 VDD_5V_SENS  ADC1 SCALE(2)
+
+# pin7 on AD&IO, analog 12
+PC2 ADC1_6V6 ADC1 SCALE(2)
+
+# pin6 on AD&IO, analog 13
+PC3 ADC1_3V3 ADC1 SCALE(1)
+
+# SPI1 - IMU3 ICM45686
+PA5 SPI1_SCK SPI1
+PB5 SPI1_MOSI SPI1
+PG9 SPI1_MISO SPI1
+PI9 SP1_CS1 CS
+
+# SPI2 -IMU1 ICM45686
+PI1 SPI2_SCK SPI2
+PI2 SPI2_MISO SPI2
+PI3 SPI2_MOSI SPI2
+PH5 SP2_CS1 CS
+PA10 SP2_DRDY2 INPUT
+
+# SPI3 -IMU2 BMI088
+PB2  SPI3_MOSI SPI3
+PC10 SPI3_SCK  SPI3
+PC11 SPI3_MISO SPI3
+PI4 SP3_CS1 CS
+PI8 SP3_CS2 CS
+PI6 SP3_DRDY1 INPUT
+PI7 SP3_DRDY2 INPUT GPIO(93)
+define SP3_DRDY2 93
+
+# SPI4 - unused
+#PE12 SPI4_SCK SPI4
+#PE13 SPI4_MISO SPI4
+#PE14 SPI4_MOSI SPI4
+#PF3  SP4_DRDY1 INPUT
+PH15 SP4_CS1 CS
+
+# SPI5 - FRAM
+PF7  SPI5_SCK  SPI5
+PH7  SPI5_MISO SPI5
+PF11 SPI5_MOSI SPI5
+PG7  FRAM_CS CS
+
+# SPI6 - external1 (disabled to save DMA channels)
+# PB3  SPI6_SCK SPI6
+# PA6  SPI6_MISO SPI6
+# PG14 SPI6_MOSI SPI6
+# PI10   EXT1_CS CS
+
+# PWM output pins
+PI0  TIM5_CH4 TIM5 PWM(1) GPIO(50)
+PH12 TIM5_CH3 TIM5 PWM(2) GPIO(51)
+PH11 TIM5_CH2 TIM5 PWM(3) GPIO(52)
+PH10 TIM5_CH1 TIM5 PWM(4) GPIO(53)
+PD13 TIM4_CH2 TIM4 PWM(5) GPIO(54)
+PD14 TIM4_CH3 TIM4 PWM(6) GPIO(55) 
+
+# Pin for PWM Voltage Selection, 0 means 3.3v, 1 means 5v
+PG6 PWM_VOLT_SEL OUTPUT HIGH GPIO(3)
+define HAL_GPIO_PWM_VOLT_PIN 3
+define HAL_GPIO_PWM_VOLT_3v3 0
+
+# we need to disable DMA on the last 2 FMU channels
+# as timer 12 doesn't have a TIMn_UP DMA option
+PH6  TIM12_CH1 TIM12 PWM(7) GPIO(56) NODMA
+PH9  TIM12_CH2 TIM12 PWM(8) GPIO(57) NODMA
+
+# GPIOs
+PE11 FMU_CAP1 INPUT GPIO(58)
+PC0 NFC_GPIO INPUT GPIO(60)
+
+# CAN bus
+PD0  CAN1_RX CAN1
+PD1  CAN1_TX CAN1
+
+PB12 CAN2_RX CAN2
+PB13 CAN2_TX CAN2
+
+# I2C buses
+
+# I2C1, GPS+MAG
+PB9 I2C1_SDA I2C1
+PB8 I2C1_SCL I2C1
+
+# I2C2, GPS2+MAG
+PF1 I2C2_SCL I2C2
+PF0 I2C2_SDA I2C2
+PG5 DRDY1_BMP388 INPUT
+
+# I2C3, MS5611, external
+PA8 I2C3_SCL I2C3
+PH8 I2C3_SDA I2C3
+
+# I2C4 internal
+PF14 I2C4_SCL I2C4
+PF15 I2C4_SDA I2C4
+
+# order of I2C buses
+I2C_ORDER I2C4 I2C1 I2C2 I2C3
+define HAL_I2C_INTERNAL_MASK 1
+
+# heater
+PB10 HEATER_EN OUTPUT LOW GPIO(80)
+define HAL_HEATER_GPIO_PIN 80
+
+# Setup the IMU heater
+define HAL_HAVE_IMU_HEATER 1
+define HAL_IMU_TEMP_DEFAULT 45
+define HAL_IMUHEAT_P_DEFAULT 50
+define HAL_IMUHEAT_I_DEFAULT 0.07
+
+# armed indication
+PE6 nARMED OUTPUT HIGH

```

Comment:
where is this pinned out/connector?

Reply:
![PE6](https://github.com/user-attachments/assets/378614fa-b180-4906-bf76-d72f7c065748)
PE6 is the nARMED label on the AD&IO terminal, which I marked in this picture.